app/controllers/bus_stops_controller.rb

```diff
@@ -53,7 +54,8 @@ def manage
   end
 
   def name_search
-    stop = BusStop.find_by name: params.require(:name)
+    name = params[:name][/[^(]+/].strip
```

Comment:
Should this also be a model method?

Reply:
Yes please.

Reply:
This is how I would make this a model method `name = BusStop.remove_hastus_id_from_name(params[:name])`. Since `params[:name]` doesn't actually belong to a BusStop, is that the right thing to do?